validators.py

```diff
@@ -0,0 +1,60 @@
+from pydantic import BaseModel, field_validator, model_validator, EmailStr
+
+
+class CryptoDict(BaseModel):
```

Comment:
Dict for this and the next is a bit redundant since classes are always always dicts